extensions/amp-carousel/0.1/carousel-controls.js

```diff
@@ -84,7 +85,10 @@ export class CarouselControls {
         event.preventDefault();
         onInteraction();
       }
-    });
+    }
+
+    addSerializedEventListener(button, 'click', onInteraction);
+    addSerializedEventListener(button, 'keydown', onKeyDown);
```

Comment:
I feel weird about substantially changing the way code/handlers are written purely for serialization purposes. Will discuss offline